charge_rate_limits.py

```diff
@@ -0,0 +1,8 @@
+CHARGE_RATE_MIN = 0.2 #assuming min limit as 0.2
```

Comment:
Good idea to define constants. Is this constant reusable at all places (production and test code) where limits are used?